### PR TITLE
SQF should have it's own preprecesssor definition

### PR DESCRIPTION
**Pull request by yyyc514**
- relying on the one from C++ caused unpredictable results
  depending on which language you would parse first because
  the pre-processor regexs would either be compiled case 
  insensative (SQF) or case-sensative (C++), depending on 
  which language you highlighted first

### CODE REVIEW

**Comment by yyyc514** [comment]
Do I need to rebase this onto master?

File: src/languages/sqf.js [hunk 1]
```diff
@@ -40,6 +40,30 @@ function(hljs) {
     ]
   };
 
+  // list of keywords from:
+  // https://community.bistudio.com/wiki/PreProcessor_Commands
+  var PREPROCESSOR =       {
```
Comment (marcoscaceres) [inline src/languages/sqf.js]:
```suggestion
  var PREPROCESSOR = {
```

**Comment by yyyc514** [comment]
Do we not care about "merge branch 'master' into whatever" commits polluting the master branch?  I really don't mind rebasing if that would help.

**Comment by marcoscaceres** [comment]
Nope, we use squash merge for PRs, so we never get merge commits in history.

**Comment by yyyc514** [comment]
Well that's useful to know, I guess now I can worry less about the exact sequencing of tiny commits for a larger PR if they are never going to see master anyways.

**Comment by egor-rogov** [comment]
Done with it. Thanks!

**Comment by yyyc514** [comment]
Thanks!

**Pull request merged**